docs/aerodromes/classd/Moorabbin.md

```diff
@@ -81,13 +116,88 @@ Aircraft who will transit Class G airspace on climb into CTA will be **cleared t
     **SGE**: "Cleared to leave and re-enter controlled airspace climbing to A050, Departures 129.4, SGE"
 
 ### Arrivals/Overfliers
-ML TCU will heads-up coordinate arrivals/overfliers from Class C to MB ADC.  
-IFR aircraft will be cleared for the coordinated approach (Instrument or Visual) prior to handoff to MB ADC, unless MB ADC nominates a restriction.  
-VFR aircraft will not be coordinated, and will need to receive their airways clearance from MB ADC.
+
+- ML TCU will heads-up coordinate arrivals/overfliers from Class C to MB ADC.  
+- **IFR arrivals** will be cleared for the coordinated approach (Instrument or Visual) prior to handoff to MB ADC, unless MB ADC nominates a restriction.  
+- **VFR arrivals** will not be coordinated, and will need to receive their airways clearance from MB ADC.
 
 !!! example
     <span class="hotline">**MDS** -> **MB ADC**</span>: "To the west, PLE, for the NDB-A"  
     <span class="hotline">**MB ADC** -> **MDS**</span>: "PLE, NDB-A"
 
 !!! tip
-    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. Use other separation methods, such as holding a departure on the ground, if separation is required with these aircraft.
\ No newline at end of file
+    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. Use other separation methods, such as holding a departure on the ground, if separation is required with these aircraft.
+
+## Helicopters
+
+### Circuits
+
+#### Day VFR Circuits
+Helicopter circuits are conducted from designated areas based on the active runway:
+
+<figure markdown>
+![Eastern Grass and Western Triangle](img/mbhelicircuitareas.png){ width="700" }
+<figcaption>Eastern Grass and Western Triangle</figcaption>
+</figure>
+
+- **RWY 17/35**: Circuits are conducted from the **Eastern Grass**.
+- **RWY 13/31**: Circuits are conducted from the **Western Triangle**.
+- Circuit operations are **parallel to the duty runway** and inside the fixed-wing circuit.
+
+**Radio Procedure**:
+- Pilots will report “AIRBORNE” before each circuit, or if departing additionally they will include their intentions. Aircraft are expected to remain under `A001` until you acknowledge.
+- Reply with simply the aircraft callsign for a leg of the circuits, or departure instructions. Hold the aircraft if they will conflict with other traffic.
+- On completion of circuit training, the pilot will nominate their landing HLS during the airborne call and report downwind before transitioning to the arrival procedure.
+
+!!! example
+    **RJB**:: "MB Gnd, RJB, At the southern apron, with information november, request circuits"
+    **MB SMC**: "RJB, cleared for circuit, Eastern Grass, report airborne"      
+    **RJB**: "Cleared for circuits, Eastern Grass, RJB"  
+    *(Before departure)*  
+    **RJB**: "Moorabbin Tower, RJB, airborne"  
+    **MB ADC**: "RJB"
```

Comment:
I'm sure this is based off the IRL procedure, but for simplicity on VATSIM, I'd support this initial clearance call being handled by ADC. Otherwise, we will need to write coordination rules for SMC to advise ADC that an aircraft is being processed for circuits. And even just a 'cleared to operate in the circuit area, not above A010, eastern grass, report airborne', for standardisation with how it's taught in the Academy.

Reply:
MB changed their real life procedures this week. Before helicopters would just report to ADC their intentions for circuits.

Now they handed it off to SMC so a slip can be made. As VATPAC doesn't deal with slips its likely less of a concern, do we want to go old school helicopter rules here?